examples/platform/efr32/OTAConfig.h

```diff
@@ -0,0 +1,32 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "app/clusters/ota-requestor/BDXDownloader.h"
+#include "app/clusters/ota-requestor/OTARequestor.h"
+#include "platform/EFR32/OTAImageProcessorImpl.h"
+#include "platform/GenericOTARequestorDriver.h"
+
+class OTAConfig
```

Comment:
should this be a namespace instead as it is used just as `OTAConfig::Init();` where init is static?

Reply:
yes it could, we'll probably change it in a follow up PR since namespace vs class isn't critical for OTA to succeed